apps/builder/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "AGPL-3.0-or-later",
   "scripts": {
     "dev": "cross-env ENVSH_ENV=.env.local sh ../../env.sh next dev -p 3000",
-    "build": "next build",
+    "build": "cross-env ENVSH_ENV=.env.local sh ../../env.sh next build",
```

Comment:
@baptisteArno this loads the env in the manual deployment but I tweak the command i.e change `sh` to `bash`
```json
"build": "cross-env ENVSH_ENV=.env.local bash ../../env.sh next build"
```

Additionally, I added to it to apps/viewer build command

Reply:
Wondering why `sh` wouldn't work? What's your OS / distrib?

Reply:
Linux / Ubuntu 22

Reply:
What error do you have when running the script with `sh`?

Reply:
Here is the output:
```bash
-e env.sh loaded
../../env.sh: 75: [[: not found
../../env.sh: 102: Syntax error: redirection unexpected
```